### PR TITLE
PR: Implement various "colour.models" changes related to "ACES VFX Enablement Working Group" work.

### PR DESCRIPTION
**Pull request by KelSolaar**

### CODE REVIEW

File: colour/models/dataset/aces.py [hunk 1]
```diff
@@ -0,0 +1,436 @@
...
+
+    Returns
+    -------
+    numeric
+        Companded value.
+    """
+
+    constants = ACES_PROXY_CONSTANTS.get(bit_depth)
+
+    float_2_cv = lambda x: max(constants.CV_min,
```
Comment (MichaelMauderer) [inline colour/models/dataset/aces.py]: Might it be clearer to use [`numpy.clip`](http://docs.scipy.org/doc/numpy/reference/generated/numpy.clip.html)?

Comment (KelSolaar) [inline colour/models/dataset/aces.py]: Yes most likely! I decided to leave it as is because it is implemented this way in the Technical Bulletin.

**Comment by MichaelMauderer** [comment]
Okay. Well it's not a major thing. 
Everything else looks fine to me.

**Comment by KelSolaar** [comment]
Ok excellent! Thanks!

**Pull request merged**